lib/games_manager.rb

```diff
@@ -87,36 +87,83 @@ def average_goals_by_season
     end
   end
 
+  def get_season_results(team_id)
+    summary = {}
+    @games.each do |game|
+      if team_id == game.away_team_id || team_id == game.home_team_id
+        summary[game.season] = [] if summary[game.season].nil?
+        if team_id == game.home_team_id && game.home_goals > game.away_goals
+          result = "W"
+        elsif team_id == game.away_team_id && game.home_goals < game.away_goals
+          result = "W"
+        elsif game.home_goals == game.away_goals
+          result = "T"
+        else
+          result = "L"
+        end
```

Comment:
Looks good. We'll have to figure out how to squish this down or use some helpers.